tests/core/pyspec/eth2spec/test/utils/randomized_block_tests.py

```diff
@@ -60,6 +62,146 @@ def _randomize_deposit_state(spec, state, stats):
     }
 
 
+def _randomize_phase0_fields(spec, state):
+    """Set Phase0-specific fields to realistic non-default values."""
+    if is_post_altair(spec):
```

Comment:
@richardgreg I think this ìf` either should not be here or throw an exception. Like it should not be allowed to call this function in the phase that is not supposed to be used. I search our code base and there are no examples like this, I think.

What do you think?